app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/account_menu.scala.html

```diff
@@ -0,0 +1,35 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.ViewConfig
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.routes
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.views.account.AccountMenuItem
+
+@this()
+
+@(activeLink: Option[AccountMenuItem])(implicit messages: Messages, appConfig: ViewConfig)
+
+<nav class="account-menu is-smaller" role="navigation">
+  <a href="@{routes.HomeController.homepage().url}" class="as-link@if(activeLink.exists(_.is[AccountMenuItem.Home])){-home }">
+    @messages("account.menu.home")
+  </a>
+  <a href="@{routes.HomeController.manageYourDetails().url}" class="as-link@if(activeLink.exists(_.is[AccountMenuItem.ManageYourDetails])){-active }">
+    @messages("account.menu.manageYourDetails")
+  </a>
+  <a href="@{appConfig.signOutUrl}?continue=/cgt-property-disposals/signed-out" class="as-link">
```

Comment:
it'd be safer to use `routes.HomeController.signedOut().url` here for the continue url

Reply:
done